### PR TITLE
fix: change "Start" stuff according to design review

### PR DESCRIPTION
**Pull request by Graborg**
**Ticket:** [#174](https://trello.com/c/jHnKDHVc/174-fixar-f%C3%B6r-start)
**Intent:** Make css changes according to trello card.
**How to test (optional):**

### All of the following commands should pass.

- [ ] `npm test`
- [ ] `npm run lint`
- [ ] `npm run cypress`

### Browser testing

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] IE11
- [ ] Edge

### CODE REVIEW

File: src/views/Start/Start.tsx [hunk 1]
```diff
@@ -29,8 +34,9 @@ const InfoBlock = styled.div`
 const Divider = styled.hr`
   background: ${({ theme }) => theme.colors.persianBlue};
   border: 0;
-  height: 2px;
+  height: 1px;
   width: 200px;
+  border-radius: 5px;
```
Comment (dubbelnisse) [inline src/views/Start/Start.tsx]: This is unnecessary, an 1 px element cant have border-radius.

File: src/views/Start/Start.tsx [hunk 2]
```diff
@@ -70,9 +76,9 @@ const Start: React.FC<RouteComponentProps> = () => {
           flexDirection="column"
           justifyContent="center"
         >
-          <img alt="logo" src={logotypeIcon} />
+          <LogoContainer alt="logo" src={logotypeIcon} />
           <Divider />
-          <Paragraph color="persianBlue" mt={0}>
+          <Paragraph color="persianBlue" fontWeight="bold" mt={0}>
             Ditt digitala CV
```
Comment (dubbelnisse) [inline src/views/Start/Start.tsx]: This is a super small detail but it should say "ditt digitala CV".

File: src/views/Start/Start.tsx [hunk 3]
```diff
@@ -70,9 +76,9 @@ const Start: React.FC<RouteComponentProps> = () => {
...
           </Paragraph>
           <Clouds alt="clouds" src={cloudsIcon} />
```
Comment (dubbelnisse) [inline src/views/Start/Start.tsx]: Based on the feedback in Trello, the illustration should be a bit smaller.

File: src/views/Start/Start.tsx (hunk 1 above)
Comment (momentiris) [reply src/views/Start/Start.tsx]: sounds weird yes. 
Zoom in and look at it...

Comment (momentiris) [reply src/views/Start/Start.tsx]: But you're right. before it was 2px that actually got round edges from border radius. but now its 1px

File: src/views/Start/Start.tsx (hunk 2 above)
Comment (momentiris) [reply src/views/Start/Start.tsx]: fixed

File: src/views/Start/Start.tsx (hunk 1 above)
Comment (momentiris) [reply src/views/Start/Start.tsx]: fixed

File: src/views/Start/Start.tsx (hunk 3 above)
Comment (momentiris) [reply src/views/Start/Start.tsx]: fixed

**Pull request merged**